assets/src/edit-story/elements/text/edit.js

```diff
@@ -269,7 +282,10 @@ function TextEdit({
       ? `${dataToEditorY(updatedFontSize)}px`
       : '';
     if (highlight) {
+      const updatedMargin = transform?.updates?.marginOffset;
       highlight.style.fontSize = target.style.fontSize;
+      highlight.style.margin = `${dataToEditorY(-updatedMargin) / 2}px 0`;
+      target.style.margin = `${dataToEditorY(-updatedMargin) / 2}px 0`;
```

Comment:
With a good mix of highlight, line-height, and margin, I can create some pretty weird boundaries here - the left margin is too small, the right and bottom ones too big (but the top one is _just right_):

![Screen Shot 2020-09-03 at 00 25 27](https://user-images.githubusercontent.com/637548/92071206-26266700-ed7c-11ea-9d67-68079e731d52.png)

Reply:
It's not yet ready for highlight+margin/padding mix, that will be solved in another PR, this one focuses on cropping the whitespace.